solr/core/src/java/org/apache/solr/handler/component/ExpandComponent.java

```diff
@@ -668,6 +676,18 @@ public void collect(int docId) throws IOException {
 
   }
 
+  private static Collector getExpandCollector(int limit, Sort sort) throws IOException {
```

Comment:
I think this method should be part of `GroupCollector` (default implementation) and method name could simply be `getCollector`

Reply:
done